ci-operator/step-registry/stackrox/deploy-acs/stackrox-deploy-acs-commands.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# !! WARNING !! There is currently a bug with single quotes in this script. It has to do with `handle_dangling_processes`
+# in the stackrox repo.
+
+# TODO(blugo): Check if these files exist
+# shellcheck source=/dev/null
+source "scripts/ci/lib.sh"
+# shellcheck source=/dev/null
+source "tests/e2e/lib.sh"
+# shellcheck source=/dev/null
+source "scripts/ci/gcp.sh"
+# shellcheck source=/dev/null
+source "scripts/ci/sensor-wait.sh"
+# shellcheck source=/dev/null
+source "scripts/ci/create-webhookserver.sh"
+# shellcheck source=/dev/null
+source "tests/scripts/setup-certs.sh"
+# shellcheck source=/dev/null
+source "qa-tests-backend/scripts/lib.sh"
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+openshift_ci_mods
+openshift_ci_import_creds
+create_exit_trap
+
+# Might not need some of these but adding for consistency for now
+export CI_JOB_NAME="ocp-qa-e2e-tests"
+export DEPLOY_STACKROX_VIA_OPERATOR="true"
+export ORCHESTRATOR_FLAVOR="openshift"
+export ROX_POSTGRES_DATASTORE="true"
+
+# Sensor will not deploy without scaling
+scripts/ci/openshift.sh scale_worker_nodes "1"
+
+gather_debug_for_cluster_under_test
+poll_for_system_test_images "3600"
+
+# Essentially replicates config_part_1
```

Comment:
Why not `qa-tests-backend/scripts/run-part-1.sh config_part_1" then? And do the `DIRECTORIES_TO_SHARE` setup after?